content/src/content/jcr_root/apps/core/wcm/sandbox/commons/ui/clientlibs/panelselect/js/polyfills.js

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
```

Comment:
Could we find a way to not duplicate the polyfills?

Reply:
@vladbailescu - for the UI panel container / selector code, I moved to use the jQuery equivalents as we still widely use jQuery there.

For the tabs site clientlib, I added the polyfills directly, as we do for the image component. Again, we don't have a common component pattern which would be the ideal place for adding these polyfills so they could be used by all components. I think we can move them there when we create such a component.